src/libs/ajax/DAR.js

```diff
@@ -16,7 +16,7 @@ export const DAR = {
     return await res.json()
   },
 
-  // v2, v3 Draft DAR Update
+  // v2, v3 Draft Progress Report
```

Comment:
I'm not sure I understand why this comment is changing.  If my memory is correct, we still have draft DARs and we don't have draft Progress Reports.